dsl/src/main/kotlin/com/chutneytesting/kotlin/junit/engine/execution/ExecutionUtil.kt

```diff
@@ -1,4 +1,4 @@
-package com.chutneytesting.kotlin.junit.engine
+package com.chutneytesting.kotlin.junit.engine.execution
```

Comment:
Not sure abour that.
This was meant to be specific to junit-engine and not shared (for the launcher for instance).
Moreover, we have to keep the engine domain dependency (which is not ideal) to junit engine only i think.